pkg/quota/registry.go

```diff
@@ -8,6 +8,11 @@ import (
 )
 
 // NewRegistry returns a registry object that knows how to evaluate quota usage of OpenShift resources.
-func NewRegistry(osClient osclient.Interface) kquota.Registry {
-	return image.NewImageRegistry(osClient)
+// If the registry will be used in admission plugin, pass true to forAdmission.
+func NewRegistry(osClient osclient.Interface, forAdmission bool) kquota.Registry {
```

Comment:
This is not obvious to me yet, but I am trying to work @pweil- comment backwards.

Upstream in Kubernetes, we did not encounter the need to have a different `Evaluator` for use in admission control than for use in the background quota controller.  The goal was to keep them unified since we should ideally be counting usage the same in both places.

I need to look elsewhere why this was needed, but it would be nice to at least comment in code why you required this split.